ModManager/MapSystem/MapManifestFinder.cs

```diff
@@ -23,14 +24,23 @@ public IEnumerable<Manifest> Find()
 
             if (!File.Exists(manifestPath))
             {
-                return new List<Manifest>();
+                return new List<MapManifest>();
             }
 
-            List<MapManifest> manifests = _persistenceService.LoadObject<List<MapManifest>>(manifestPath, false);
-
-            UpdateManifestInfo(manifests);
-
-            return manifests;
+            try
+            {
+                List<MapManifest> manifests = _persistenceService.LoadObject<List<MapManifest>>(manifestPath, false);
+                UpdateManifestInfo(manifests);
+                return manifests;
+            }
+            catch (JsonSerializationException ex)
+            {
+                return new List<MapManifest>();
```

Comment:
Add a log here with just the message, maybe some info text how to fix it

Reply:
Added